pom.xml

```diff
@@ -81,6 +81,7 @@
         <!-- Sonarcloud properties -->
 
         <version.slf4j>1.7.30</version.slf4j>
+        <version.software-amazon-awssdk>2.14.12</version.software-amazon-awssdk>
         <version.aws-java-sdk-s3>1.11.850</version.aws-java-sdk-s3>
```

Comment:
Just for my better understanding: Will we still be needing this property + dependency? Are they meant to be used together, or will this be dropped as well?

Reply:
No, this `aws-java-sdk-s3` dependency will be dropped as result of the migration.